content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Join the DVC Ambassador Program!'
+date: 2020-05-08
+description: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+picture: 2020-05-08/Ambassador_Header.png
+author: marcel_rd
+commentsUrl: https://discuss.dvc.org/t/new-video-reimagining-devops-for-machine-learning/341
+tags:
+  - Ambassador
+  - Volunteer
+  - Talks
+  - Meetup
+  - Outreach
+---
+
+Software can be everywhere, but developers can’t - that’s why ambassadors, folks
```

Comment:
This phrase looks very good but it was not clear to me when I read it for the first time. Why developers are not everywhere? They are. Oh... you are talking about _your_ developers. Ok...

Is there a way to clarify it? Ok, if not.

Reply:
The idea is that softwares can be infinite (infinite copies), while developers are a limited number of people doing code. Ambassadors, extra people, come in to try to increase the number of people related to the project through which you can interact.

The idea was to have a catchy phrase, and usually, these phrases are short, though not always easy to get the idea at first sight :-|. I agree with you, it's not very clear.

What do you think of:

Software can be everywhere at the same time, but developers can't -

?

Reply:
changed to `DVC's software can be everywhere, but its developers can’t`